app/code/Magento/Catalog/etc/adminhtml/system.xml

```diff
@@ -138,6 +138,12 @@
                     <hide_in_single_store_mode>1</hide_in_single_store_mode>
                 </field>
             </group>
+            <group id="layered_navigation">
+                <field id="display_category" translate="label" type="select" sortOrder="15" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Display Category</label>
```

Comment:
I would clarify the title of the current option and name it as "Display Category Filter", what do you think?